src/encoded/static/components/filegallery.js

```diff
@@ -463,6 +463,18 @@ export class FileTable extends React.Component {
                 return analysisObjectsAccession || nonAnalysisObjectPrefix;
             });
 
+            const elementsReferences = context.elements_references;
+
+            if (elementsReferences) {
+                const elementsReferenceFiles = elementsReferences.map((elementsReference) => elementsReference.files).reduce((acc, val) => acc.concat(val), []);
```

Comment:
I see what you’re doing, though I think it’d be easier to understand in one step:
```
const elementsReferenceFiles = elementsReferences.reduce((accFiles, reference) => accFiles.concat(reference.files));
```